docs/technical_reference/flowsheets/RO_with_energy_recovery.rst

```diff
@@ -0,0 +1,162 @@
+    :align: center
+
+    Figure 1: Process flow for RO with pressure exchanger ERD.
+
+Figure 2 presents the process flow diagram if ``erd_type == pump_as_turbine``.
+
+.. figure:: ../../_static/flowsheets/RO_w_ERD-turbine.png
+    :width: 600
+    :align: center
+
+    Figure 2: Process flow for RO with pump-as-turbine ERD.
+
+
+Figure 3 presents the process flow diagram with no ERD (i.e., ``erd_type == no_ERD``).
+
+.. figure:: ../../_static/flowsheets/RO_w_ERD-no_ERD.png
+    :width: 600
+    :align: center
+
+    Figure 3: Process flow for RO without ERD.
+
+
+Degrees of Freedom 
+------------------
+
+The degrees of freedom (DOF) for the flowsheet can change depending on model configuration options.
+For either ``pump_as_turbine`` or ``pressure_exchanger`` as ``erd_type``, there are 15 DOF. Running
+the model with ``no_ERD`` results in 13 DOF.
+
+* Influent conditions (component flows, temperature, pressure)
+* RO membrane properties
+* RO operating pressure
+* Pump and ERD efficiencies
+
+Passing any model build to the provided function ``set_operating_conditions()`` will result in a model with zero DOF.
+
+
+Flowsheet Specifications
+------------------------
+
+The influent and operating conditions for the different flowsheet configurations are presented in the following table,
+including the different build options for ``erd_type``:
+
+.. csv-table::
+   :header: "Description", "Default Value", "Units"
+
+    **Influent Conditions**
+   "Volumetric flow rate", "1e-3", ":math:`\text{m}^3/\text{s}`"
+   "TDS mass fraction", "0.035", ":math:`\text{dimensionless}`"
+   "Temperature", "298", ":math:`\text{K}`"
+   "Pressure", "101325", ":math:`\text{Pa}`"
+   
+   **Desalination**
+   "RO water permeability coefficient", "4.2e-12", ":math:`\text{m/Pa/s}`"
+   "RO salt permeability coefficient", "3.5e-8", ":math:`\text{m/s}`"
+   "RO spacer porosity", "0.85", ":math:`\text{dimensionless}`"
+   "RO channel height", "1e-3", ":math:`\text{m}`"
+   "RO membrane width per stage", "5", ":math:`\text{m}`"
+   "RO total membrane area per stage", "50", ":math:`\text{m}^2`"
+   "RO permeate side pressure", "101325", ":math:`\text{Pa}`"
+   "Pump 1 efficiency", "0.8", ":math:`\text{dimensionless}`"
+   "Pump 1 operating pressure", "70e5", ":math:`\text{Pa}`"
+   
+   *if* ``erd_type == "pressure_exchanger"``
+   "Pressure exchanger efficiency", "0.95", ":math:`\text{dimensionless}`"
+   "Pump 2 efficiency", "0.8", ":math:`\text{dimensionless}`"
+   
+   *if* ``erd_type == "pump_as_turbine"``
+   "Energy recovery device pump efficiency", "0.95", ":math:`\text{dimensionless}`"
+   "Energy recovery device permeate side pressure", "101325", ":math:`\text{Pa}`"
+   
+
+Code Documentation
+------------------
+
+* :mod:`watertap.examples.flowsheets.RO_with_energy_recovery`
+
+References
+----------
+
+References for each component and model used in this flowsheet can be found using the links provided in the Implementation section of this documentation.
```

Comment:
I think we can just delete this section. References should be reserved for literature references.